content/tutorial-deep-learning-on-mnist.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<p>The numbering for the items in this list seems to be off </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
The rendering here is strange too, because I'm using `3. ... - _Learning rate_... - _Epochs (...)_` I added tabs, just in case:

```diff
3. Set certain default values of [hyperparameters](https://en.wikipedia.org/wiki/Hyperparameter_(machine_learning)), such as:
- - [_Learning rate_](https://en.wikipedia.org/wiki/Learning_rate): ...
+     - [_Learning rate_](https://en.wikipedia.org/wiki/Learning_rate): ...
...
```